.gitignore

```diff
@@ -6,3 +6,6 @@ dist
 .coverage.*
 .coverage
 .tox
+.*cache
```

Comment:
The projects policy is that such things belong in your global gitignore, rather than us ending up with everything in ours.